client/homebrew/brewRenderer/toolBar/toolBar.jsx

```diff
@@ -32,16 +34,16 @@ const ToolBar = ({ displayOptions, currentPage, totalPages, onDisplayOptionsChan
 			setPageNum(parseInt(pageInput)); // input type is 'text', so `page` comes in as a string, not number.
 	};
 
+	// scroll to a page, used in the Prev/Next Page buttons.
```

Comment:
Not sure if this comment actually adds anything

Reply:
I think as I wrote it i was just erring on the side of "too many comments" rather than too few.  Just before writing this reply i was reading a blog post that had this bit in it:

> You know how wired earbuds always, always get tangled when you place them in a drawer or your pocket or something for few seconds? Codebases do the exact same thing, when there are multiple people writing on things, fixing each others' bugs. Everyone has a different way of thinking, so it's only natural that things over time get a bit tangled up.
> ...
> When going through old code, especially some that has barely any comments, it can take a very long time to understand what is actually going on. I honestly spent most of my time trying to understand how the thing even works, what is called when, where the icons positions are updated, and so on.

I don't know if it's worth keeping this comment in or not, but right now, if it were me, I would.  I don't think it's bad to have a comment above each function describing what it does and where it's called even if it's pretty obvious to us right now what it does.

Someone else can remove it or resolve the conversation however they see fit.